main.go

```diff
@@ -120,6 +149,41 @@ func main() {
 		setupLog.Error(err, "unable to create controller", "controller", "Gatekeeper")
 		os.Exit(1)
 	}
+
+	var wg sync.WaitGroup
+
+	dynamicClient := dynamic.NewForConfigOrDie(mgr.GetConfig())
+	wg.Add(1)
+
+	c := ConstraintStatusInstaller{isInstalled: false}
```

Comment:
If you follow my other comments, you won't need a struct anymore. A function will suffice.